facts/4.2/redhat-7-x86_64.facts

```diff
@@ -1,204 +1,78 @@
 {
-  "aio_agent_version": "7.9.0",
-  "architecture": "x86_64",
-  "augeas": {
-    "version": "1.12.0"
-  },
-  "augeasversion": "1.12.0",
-  "bios_release_date": "12/01/2006",
-  "bios_vendor": "innotek GmbH",
-  "bios_version": "VirtualBox",
-  "blockdevice_sda_model": "VBOX HARDDISK",
-  "blockdevice_sda_size": 42949672960,
-  "blockdevice_sda_vendor": "ATA",
-  "blockdevices": "sda",
-  "boardmanufacturer": "Oracle Corporation",
-  "boardproductname": "VirtualBox",
-  "boardserialnumber": "0",
-  "chassistype": "Other",
-  "dhcp_servers": {
-    "eth0": "10.0.2.2",
-    "eth1": "192.168.56.100",
-    "system": "10.0.2.2"
-  },
-  "disks": {
-    "sda": {
-      "model": "VBOX HARDDISK",
-      "size": "40.00 GiB",
-      "size_bytes": 42949672960,
-      "type": "hdd",
-      "vendor": "ATA"
-    }
-  },
-  "dmi": {
-    "bios": {
-      "release_date": "12/01/2006",
-      "vendor": "innotek GmbH",
-      "version": "VirtualBox"
-    },
-    "board": {
-      "manufacturer": "Oracle Corporation",
-      "product": "VirtualBox",
-      "serial_number": "0"
+  "os": {
```

Comment:
Ideally the facts would be sorted. You can use `jq --sort-keys < facts/4.2/redhat-7-x86_64.facts` to do so.

Reply:
Should now be fixed. Great that the automatic checks looks for files with the same information in them.